app/views/adoption_requests/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h2>Adoption Requests</h2>
+<% @gems.each do |g| %>
+	<h3><%= g.gem %></h3>
+	<ul>
+		<% g.adoption_requests.order('created_at DESC').each do |r| %>
+			<li>
+				<p><strong><%= current_user.username %></strong>: <%= r.description %></p>
+				<button>Accept</button>
+			</li>
+		<% end %>
+	</ul>
+<% end %>
```

Comment:
Remember to set use double space instead of tabs, it's just a convention in the Ruby community.

What text editor are you using, I might be able to help you set that up.